hs_core/models.py

```diff
@@ -270,7 +270,7 @@ def update(cls, element_id, **kwargs):
         if 'description' in kwargs:
             party = cls.objects.get(id=element_id)
             if party.description is not None and kwargs['description'] is not None:
-                if len(party.description) > 0 and len(kwargs['description']) > 0:
+                if len(party.description.strip()) > 0 and len(kwargs['description'].strip()) > 0:
                     if party.description != kwargs['description']:
```

Comment:
What I mean by the other way is that you may not raise validation error if they only differ by white spaces at the beginning and end which will satisfy your comparison condition here, but instead, you just return without updating them since they are "equal" after stripping out white spaces, so there is no need for updating. Hope this makes sense. Since this is a very special case and not critical at all, I am OK with the way you are doing now, but think the other way as I explained here also makes sense for consideration.